landing-pages/site/content/en/blog/announcing-new-website.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "New Airflow website"
+linkTitle: "New Airflow website"
+author: "Aizhamal Nurmamat kyzy"
+description: "We are thrilled about our new website!"
+tags: ["Community"]
+date: "2019-12-11"
+---
+
+The brand [new Airflow website](https://airflow.apache.org/) has arrived! Those who have been following the process know that the journey to update [the old Airflow website](https://airflow.readthedocs.io/en/1.10.6/) started at the beginning of the year. 
+Thanks to sponsorship from the Cloud Composer team at Google that allowed to 
+collaborate with Polidea and deliver an awesome website.
```

Comment:
```suggestion
collaborate with [Polidea](https://www.polidea.com/) and with their design studio [Utilo](https://utilodesign.com/), and deliver an awesome website.
```

Reply:
I forwarded this text to our PR department and they had only one suggestion.

Reply:
Marta, Agnieszka, Magda is working in Utilo. Kamill Gabriel, Zuzzana is working for Polidea.

Reply:
I think it is worth mentioning only Polidea, since the contract was directly with Polidea. What do you think?

Reply:
Utilo has no legal entity.  Each contract will be signed by Polidea. Still, because of marketing, it is challenging to sell Polidea as a software house and as a design studio, so and Polidea decided to create a new brand - Utilo.  When we describe our graphic works, we would like to be called Utilo, but if we deal with programming, we would like to be called Polidea.

At the bottom of our Utilo page we have information that this is one legal entity.
<img width="475" alt="Screenshot 2019-12-17 at 23 59 40" src="https://user-images.githubusercontent.com/12058428/71041414-4f895f00-2129-11ea-8f5a-e6f17e95bf1b.png">

I know it is difficult, but this is the structure of our company and the way our marketing works.  If anyone wanted graphic designers, they were looking for graphic designers and not looking for a software house. Hence, a division had to arise.